base/src/main/scala/co/uproot/abandon/Config.scala

```diff
@@ -204,9 +217,29 @@ case class NegativeConstraint(val accName: String) extends Constraint with SignC
   val signStr = "negative"
 }
 
+case class DateConstraint(dateFrom: Option[LocalDate], dateTo: Option[LocalDate]) extends Constraint {
```

Comment:
Nice to see that you made the from and to dates an `Option[T]`.

Should we flag the case when both `dateFrom` and `dateTo` are `None`? If so, it should be done in the config parser.